.github/workflows/release.yml

```diff
@@ -7,18 +7,18 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: 12
+          node-version: '12'
```

Comment:
Does it mean it will only work on node version 12 now on? If not, maybe it would be better to test both versions.

Reply:
Should we update the package.json as well for the minimum supported version now? @dplewis @mtrezza thoughts?

Reply:
What are the consequences of switching to yarn? I think we'd want to keep the CI of all repos as uniform as possible. Not only when thinking about mono-repo, but also release automation.

Reply:
I just changed integer to string as per specification, and in the ci task I changed 10 to 12 so it's consistent in both files. There's nothing that I know of that requires node 12 to be honest. I think it will work fine with 10. So - for your consideration.
Regarding yarn - while it's faster and more capable, the reason I switched to yarn is that npm just failed to resolve all the dependencies for some reason. After struggling with it I gave up and switched to yarn.

Reply:
Thanks I actually opened another [thread](https://github.com/parse-community/parse-dashboard/pull/1677#discussion_r608163725) to discuss yarn, to keep this thread about Node 10.

Reply:
In the current codebase, package.json, ci and release workflows all have different versions specified. Decide upon some version (I think 10 is ok) and I'll change it in all three files.

Reply:
I reverted all my changes to these files.